src/cdk/table/sticky-styler.ts

```diff
@@ -64,6 +67,7 @@ export class StickyStyler {
     private _isBrowser = true,
     private readonly _needsPositionStickyOnElement = true,
     private readonly _positionListener?: StickyPositioningListener,
+    private readonly _tableInjector?: Injector,
```

Comment:
can you just get ahold of this with `inject()` instead? then it wouldn't have to be optional and we could get rid of the logic branch for if it isn't provided

Reply:
I don't think this class is constructed in an injection context though? The table constructs this during ngOnInit